docs/3.x.x/en/advanced/customize-admin.md

```diff
@@ -62,6 +62,8 @@ The panel will be available through [http://localhost:1337/dashboard](http://loc
 
 ### Development mode
 
+Note that to modify the administration panel, your project needs to be created with using the `dev` flag, an example of such would be: `strapi new strapi --dev`.
```

Comment:
But why?. I saw the `dev` flag creates a syslink to a global package. I'm thinking that a user wants the modifications as part of their project... and this doesn't sound like that would happends

Reply:
I agree with @maturano, this `--dev` options is present in `CONTRIBUTING.md` file, because this way you work on strapi project, not on your own projet.

Reply:
@maturano without the `--dev` it's impossible to start the frontend server therefore it's really hard to see what you're developing...

Currently, the only way to include modifications into your project is to generate a new app with that includes the modifications made and copy/paste the corresponding folder into your current project.

Reply:
I start the frontend running `npm run setup` in projects without the `--dev` flag... or not? 🤔 

I have no clear the workflow for development in a project having strapi as dependency and I think is a common misconception; I did find several conversations/issues with the same doubt (maybe because I was searching about that) and with no clear answer.

Hoping improve the documentation in that matter U_U

Reply:
Back to my first comment, when I said "but why?" I was refer about that; the clarification of how the things works, an explanation of what is the idea behind, share how is it thought that things work